Surgery tweak

florkeger opened the pull request:
**What does this PR do:**
This PR is specifically aimed at making the life of surgeons a little bit less painful. **Please keep in mind, this is not a start on a medbay re-work, that's not my intention.** This PR is merely created for the sake of moving certain objects to a different location in order to eliminate a few obstacles doctors come caross quite frequently.

Please view the changelog below for what was added/changed.

**I'd appreciate sincere feedback. I know this PR isn't a lot. Some would say all I did was flipping around the objects inside the respective rooms, and you aren't too far off from that. These changes are merely Quality of Life improvements with the goal of aiding surgeons a little. I'm always up with discussion changes.**

A current discussion that came up on Discord was the pre-loaded mounted defibrillators inside each OR, whether or not they are needed or if they take away too much of the prep work. Some appreciated them, but it can all be discussed.

Picture of the ORs now: https://i.imgur.com/RqgYfQw.png

**Changelog:**
:cl: kazboo
add: added a loaded wall mounted defibrillator in each OR for easier access and technically infinite usage in the case a patient dies mid surgery
add: added a surgeon-special locker with surgeon essential equipment (scrubs, spaceacellin, sterile mask)
add: added an organ freezer for organ storage to both ORs
add: added a surgery dufflebag that comes pre-loaded with all neccessary tools to both ORs
imageadd: added a "WASH" sign
tweak: changed the outside hallways appearance by changing up the floor a little
tweak: changed the ORs windows and door location allowing for faster entering and exitting to the surgery table
tweak: moved the shutter/light/holosign buttons closer near the OR tables so they can be reached from right behind the OR table
tweak: moved the right OR disposal unit into storage
tweak: re-alligned cables, pipes and vents across both ORs and the hallway
/:cl:

Markolie commented:
`balance: changed the cameras in both the vault and Captains Office for motion sensor ones`

Mmhhhm?

florkeger commented:
> `balance: changed the cameras in both the vault and Captains Office for motion sensor ones`
> 
> Mmhhhm?

Yessssss.

That was one of the balance things I brought up at some point in Discord. Made no sense to me that rooms like those have poor surveillance especially since their high priority targets, so I just changed them up.

Markolie commented:
I'd suggest moving that into a separate PR. That really doesn't belong in this one (and it's also the one change in this pull request I oppose). The way it's added as an aside in the changelog is also not really the proper way of informing people that this PR includes that chang.e

florkeger commented:
> I'd suggest moving that into a separate PR. That really doesn't belong in this one (and it's also the one change in this pull request I oppose). The way it's added as an aside in the changelog is also not really the proper way of informing people that this PR includes that chang.e

It's more of a Valve approach I suppose. They also have a tendency of just throwing stuff out there that isn't particularly relevant to the changelog, but I personally didn't find it neccessary to make a whole new PR for simply switching out cameras. I guess I can do it as seperate, but at the time, I just didn't feel the need.

Any other feedback?

Markolie commented:
Nah, just move that one into a separate PR and it's all gucci.

Couls commented:
You made it so body scanners have no density and you can walk over them, should document that

Dave-TH commented:
Motions cameras are still in the diff

Dave-TH commented on _maps/map_files/cyberiad/cyberiad.dmm:
```diff
@@ -27535,6 +27530,10 @@
 	pixel_x = -28
 	},
 /obj/item/storage/belt/champion,
+/obj/machinery/camera/motion{
```
Here

Dave-TH commented on _maps/map_files/cyberiad/cyberiad.dmm:
```diff
@@ -43646,12 +43639,11 @@
-/obj/machinery/camera{
-	c_tag = "Captain's Office";
+/obj/item/storage/lockbox/medal,
+/obj/machinery/camera/motion{
```
And here

florkeger commented:
So turns out I somehow managed to broke my files and the entire thing is, once again, corrupted. Multiple attempts in somehow getting it back failed so far, so I'll either have to wait until this gets (hopefully) merged or I have to re-do the project - for the third time - from scratch.

Sorry folks, can't make any other changes to this, least in no way I can figure out.

Markolie commented:
You should re-do these changes on a new branch. With the motion cameras still in this is very unlikely to go through.

florkeger commented:
> You should re-do these changes on a new branch. With the motion cameras still in this is very unlikely to go through.

There you go, reverted them again, now it should be just the medbay without the cameras.

Dave-TH commented:
Cameras look good now 👍

AdvnetureDrake commented:
I think that both the surgery tray and glass tables should be within reach of each other and the surgery table. It's a minor nitpick, but it'd be a bit annoying having to move back and forth from the tray to the glass table. Perhaps move the scanner over next to the window and put the glass table directly behind where the surgeon would stand.

florkeger commented:
> I think that both the surgery tray and glass tables should be within reach of each other and the surgery table. It's a minor nitpick, but it'd be a bit annoying having to move back and forth from the tray to the glass table. Perhaps move the scanner over next to the window and put the glass table directly behind where the surgeon would stand.

While I appreciate your feedback...

https://i.imgur.com/E8C1Ccf.png

AdvnetureDrake commented:
Ah, didn't see that. Sorry.

Fox-McCloud commented:
I really don't like this. The current OR's are fine, in my opinion, except the fact the one on the left doesn't have a disposal.

Closing this due to lack of updates and also the users ongoing corruption problems.
